chart/values.yaml

```diff
@@ -631,3 +631,12 @@ rabbitmq:
         dest-uri: {{ $shovel.destUri | default "amqp://" | quote }}
         reconnect-delay: {{ $shovel.reconnectDelay | default 5 }}
     {{- end }}
+
+cert-manager:
```

Comment:
in the private repo we use `certManager`; would be nice to adjust that as well to have it in-line.